tests/test_translator.py

```diff
@@ -68,12 +71,37 @@ def test_track_to_mpd_format_for_nonempty_track(self):
         )
         assert ("file", "à uri") in result
         assert ("Time", 137) in result
+        assert ("Artist", "an artist;yet another artist") in result
```

Comment:
Looking at the failing tests on CI (which succeed on my machine), the order seems inconsistent. I think it would be fine to require that an `artist` tag with either `"an artist;yet another artist"` or `"yet another artist;an artist"` as its value is contained in `result`.

@djmattyg007, what do you think? And should the same be done for `composer` and `performer`?

Reply:
Ah, the `track.artists`, `track.album.artists`, `track.performers` and `track.composers` attributes are `frozenset`s, which are unordered, so I think this (one of the orders is contained in `result`) is indeed the proper assertion.